index.js

```diff
@@ -0,0 +1,149 @@
+    <div class="card-img">
+      <img src="${projectInfo[i].popupPhoto}" width="100%" />
+    </div>
+    <div>
+      <h2 class="card-header">${projectInfo[i].header}</h2>
+      <ul class="card-sub">
+        <li class="remove">${projectInfo[i].canopy}</li>
+        <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+        <li>${projectInfo[i].backend}</li>
+        <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+        <li>${projectInfo[i].year}</li>
+      </ul>
+      <div class="card-content">
+        <p>${projectInfo[i].content}</p>
+        <ul class="tools">
+          ${toolsHtml}
+        </ul>
+        <button class="popBtn show">See project</button>
+      </div>
+    </div>
+  `;
+
+  projects.appendChild(card);
+}
+
+for (let i = 0; i < showButton.length; i += 1) {
+  showButton[i].addEventListener('click', () => {
+    body.style.display = 'block';
+    document.body.scrollTop = 0;
+    document.documentElement.scrollTop = 0;
+
+    let toolsHtml = '';
+    for (let j = 0; j < projectInfo[i].tools.length; j += 1) {
+      toolsHtml += `<li>${projectInfo[i].tools[j]}</li>`;
+    }
+
+    popUp.innerHTML = `
+        <section id="modal-section">
+        <div class="modal-body">
+          <div class="modal-title">
+            <h2 class="card-header">${projectInfo[i].header}</h2>
+            <i id="modal-close" class="uil uil-multiply"><img src="media/cross.png"></i>
+            <ul class="card-sub">
+              <li class="remove">${projectInfo[i].canopy}</li>
+              <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+
+              <li>${projectInfo[i].backend}</li>
+              <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+
+              <li>${projectInfo[i].year}</li>
+            </ul>
+          </div>
+          <img class="image-img" src="${projectInfo[i].popupPhoto}" />       
+          <div>
+            <div class="modal-content">
+              <p>${projectInfo[i].content}</p>
+              <div class="modal-links-content">
+                <ul class="tools mobile-category">
+                  ${toolsHtml}
+                </ul>
+                <div>
+                  <hr class="line-break" />
+                  <div class="modal-combine">
+                    <button class="modal-button live">
+                      See live
+                      <img class="svg" src="media/Live-Icon.svg" alt="Live Icon" />
+                    </button>
+                    <button class="modal-button source">
+                      See source
+                      <span>
+                      <img class="svg" src="media/github.svg" alt="Github Icon" />
+                      </span>
+                      </button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          </section>
```

Comment:
- [ ] Your mobile menu popup is not there yet. Kindly follow the figma. Please check the screenshots below

**Figma**
![popup1](https://github.com/rbd3/Narson-Portfolio/assets/99656687/e2c7cd73-c74d-498f-bc7f-fe5779c92a96)

**Your implementation**
![popup](https://github.com/rbd3/Narson-Portfolio/assets/99656687/f1321413-4392-4307-80ff-bb9c7b3b558d)

- [ ] Kindly lets have this fixed 🤝